app.dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ruby:2.5
```

Comment:
Let's call this file `Dockerfile.dev`

Reply:
What is the purpose of having a separate dockerfile? What is the difference between this file and regular Dockerfile?

Reply:
Live reload code and preset chrome headless for capybara tests into docker.

Reply:
Okay, then just rename it and add a screenshot please